roles/smallfile-bench/templates/workload_job.yml.j2

```diff
@@ -40,24 +40,23 @@ spec:
           - name: uuid
             value: "{{ uuid }}"
           - name: test_user
-            value: "{{ test_user }}"
+            value: "{{ test_user | default("ripsaw") }}"
           - name: clustername
             value: "{{ clustername }}"
-{% if elasticsearch.server is defined %}
           - name: es
-            value: "{{ elasticsearch.server }}"
+            value: "{{ elasticsearch.server | default() }}"
```

Comment:
same comment as with fs-drift - did you mean "if elasticsearch is defined"?   because it doesn't make sense to have a default for elasticsearch.server if the onlly way you can get to it is if the user defines elasticsearch.server in their CR.